build-container/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e
+
+if [ ! -z "${GPG_KEY_DATA-}" ]; then
+    if [ -z "${GPG_KEY_ID-}" ]; then
+        echo 'GPG_KEY_ID is not set, but GPG_KEY_DATA is set. Please set GPG_KEY_ID to the key ID of the key.'
+        exit 1
+    fi
+    echo "${GPG_KEY_DATA}" | gpg --import /dev/stdin
+fi
+
+# Only set -x here so we can't accidently print the GPG key up there
+set -x
+
+sudo chown -R buildbot:buildbot /src
+cd /src
+
+sed -i 's/jan/buildbot/' conf.sh && sudo ccm64 d || true
```

Comment:
Why not change it in `conf.sh` outright?

Reply:
Not sure, this is taken from archzfs-ci. We could just change the default, yeah